gen/dcos-config.yaml

```diff
@@ -974,6 +974,17 @@ package:
               },
               {
                   "Command": ["systemctl", "list-units", "dcos*"]
+              },
+              {
+                  "Command": ["iptables-save"]
+              },
+              {
+               "Command": ["/bin/sh", "-c", "cat /proc/`pgrep mesos-master`/environ"],
```

Comment:
this is IMO a bit optimistic. We should better ask systemctl for the units PID file as this would be the source of truth

e.g.
```
"/bin/sh", "-c", "cat /proc/`systemctl show dcos-mesos-master.service -p MainPID| cut -d'=' -f2`/environ"
```